app/partial-body.html

```diff
@@ -1,6 +1,7 @@
 <div id="app-content">
   <img class="loading-logo" src="./images/logo/metamask-fox.svg" alt="" loading="lazy" />
   <img class="loading-spinner" src="./images/spinner.gif" alt="" loading="lazy" />
+  <div class="loading-timeout-message" aria-live="polite" role="status"></div>
```

Comment:
`aria-live="polite"` makes screen readers read this when if the content within the element updates, `role=status` is just recommended semantics and didn't change the behavior in my local testing.